lib/rules/at-rule-no-unknown/__tests__/index.js

```diff
@@ -91,6 +91,9 @@ testRule({
 		{
 			code: '@layer framework { h1 { background: white; } }',
 		},
+		{
+			code: '@scroll-timeline spin-slash { }',
```

Comment:
https://github.com/stylelint/stylelint/blob/797381ced3190aa0c78bff8fb892572caadf4305/docs/developer-guide/rules.md?plain=1#L167

Reply:
Thanks - just updated the test case!